data/json/monsters/insect_spider.json

```diff
@@ -2969,5 +2969,379 @@
     "anger_triggers": [ "PLAYER_WEAK" ],
     "fear_triggers": [ "FIRE", "PLAYER_CLOSE" ],
     "flags": [ "SEES", "HEARS", "SMELLS", "SWIMS" ]
+  },
+  {
+    "id": "mon_butterfly",
+    "type": "MONSTER",
+    "name": { "str": "mutated butterfly", "str_pl": "mutated butterflies" },
+    "description": "An oversized butterfly with vibrant colored markings.  Unlike the other big insects of the Cataclysm this is a very peaceful nectivore.",
+    "bodytype": "flying insect",
+    "species": [ "INSECT_FLYING" ],
+    "default_faction": "insect",
+    "material": [ "iflesh" ],
+    "symbol": "l",
+    "color": "white",
+    "volume": "15 L",
+    "weight": "20 kg",
+    "hp": 25,
+    "speed": 95,
+    "vision_day": 15,
+    "dodge": 2,
+    "harvest": "arachnid_firefly",
+    "reproduction": { "baby_egg": "egg_butterfly", "baby_count": 1, "baby_timer": 15 },
+    "upgrades": { "half_life": 14, "into": "mon_butterfly_giant" },
```

Comment:
I'd either use age_grow with a long timer, or let reproduction/new spawns take care of the size progression. A mix of sizes makes for better flavor than "oops all emperors".

Reply:
done, changed to age growth 30 days